src/common/evaluation/QA/eval_depthmap_models/src/utils.py

```diff
@@ -329,10 +329,18 @@ def draw_stunting_diagnosis(df: pd.DataFrame, png_out_fpath: str):
         else:
             not_processed_data.append(row['qrcode'])
     data = confusion_matrix(actual_stunting, predicted_stunting)
-    plt.rcParams.update({'font.size': 8})
+    T1, FP1, FP2, FN1, T2, FP3, FN2, FN3, T3 = data.ravel()
+    sum = T1 + FP1 + FP2 + FN1 + T2 + FP3 + FN2 + FN3 + T3
+    T = ((T1 + T2 + T3) / sum) * 100
+    FP = ((FP1 + FP2 + FP3) / sum) * 100
+    FN = ((FN1 + FN2 + FN3) / sum) * 100
+    fig = plt.figure(figsize=(15, 15))
+    ax = fig.add_subplot(111)
     disp = ConfusionMatrixDisplay(confusion_matrix=data, display_labels=STUNTING_DIAGNOSIS)
-    disp.plot(cmap='Blues', values_format='d')
-    plt.title("Stunting Diagnosis")
+    disp.plot(cmap='Blues', values_format='d', ax=ax)
+    s = "True:-" + str(round(T, 2)) + " False Positive:-" + str(round(FP, 2)) + " False Negative:-" + str(round(FN, 2))
```

Comment:
There should not be a minus sign. The numbers are positive

```suggestion
    s = "True:" + str(round(T, 2)) + " False Positive:" + str(round(FP, 2)) + " False Negative:" + str(round(FN, 2))
```